Add Company Name

xyan12345 opened the pull request.

idmontie committed:
Add visual test

xyan12345 committed:
Add Company Name

idmontie committed:
Merge branch 'develop' into xuefeng-develop

idmontie commented on client/components/game/company-name/companyName.jsx:
```diff
@@ -0,0 +1,11 @@
+export default React.createClass({
+  render() {
+    return (
+      <div className="company_name">
```
The class name should be `company-name`.

idmontie commented on client/components/game/company-name/companyName.jsx:
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Change the file name to `company-name.jsx`

idmontie commented on client/components/game/company-name/companyName.jsx:
```diff
@@ -0,0 +1,11 @@
+  render() {
+    return (
+      <div className="company_name">
+        <h>{this.props.CompanyName}</h>
```
Change the prop to `companyName`.

xyan12345 replied:
Do i just call
<h>{this.companyName}</h>?
Is that how to fix it?

idmontie replied:
`{this.props.companyName}`

idmontie commented on client/index.jsx:
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import {default as Game} from './containers/game/game.jsx';
 
+
```
Why is this extra line here?

xyan12345 committed:
change company name

xyan12345 committed:
add test of Company-name

xyan12345 committed:
Update things
erge branch 'ivan-develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop

xyan12345 committed:
Merge branch 'ivan-develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-de

efeng-develop

xyan12345 committed:
Merge branch 'develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop

update things

Update things

xyan12345 committed:
Merge branch 'develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop
Update things

xyan12345 committed:
fix company name

xyan12345 committed:
test

xyan12345 committed:
update

xyan12345 committed:
updateMerge branch 'mark-develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop

xyan12345 committed:
update
Merge branch 'develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop

xyan12345 committed:
Update works Merge branch 'xuefeng-develop' of https://github.com/gios-asu/dont-get-fired-game into xuefeng-develop

idmontie commented on client/actions/actions.js:
```diff
@@ -3,27 +3,38 @@ export const ACTION_ENUM = {
   return { type: ACTION_ENUM.NEW_GAME };
 }
 
+<<<<<<< HEAD
```
Fix this

idmontie commented on client/components/game/program-list-item/program-list-item.jsx:
```diff
@@ -6,7 +6,12 @@ import { selectProgram } from '../../../actions/actions';
 export default React.createClass({
   onClick( e ) {
     e.preventDefault();
+<<<<<<< HEAD
```
Fix merge

xyan12345 committed:
fix problems

xyan12345 committed:
fix issue

xyan12345 committed:
fix problem

idmontie commented on client/components/game/program-list-item/program-list-item.jsx:
```diff
@@ -6,7 +6,10 @@ import { selectProgram } from '../../../actions/actions';
 export default React.createClass({
   onClick( e ) {
     e.preventDefault();
+    dispatch( selectProgram(this.props.programName, this.props.credits));
```
What happened here?

idmontie commented:
This looks good except for the program-list-item onClick action

xyan12345 committed:
Modified the program-list

xyan12345 committed:
changed program-list-item

xyan12345 committed:
Add Main Side Bar

xyan12345 committed:
MainSideBar fixed

rraub committed:
Merge branch 'develop' into xuefeng-develop
